docs/telemetry.md

```diff
@@ -0,0 +1,102 @@
+# Telemetry
```

Comment:
V tomto PR je dokumentacia k telemtrii v [telemetry.md](https://github.com/keboola/keboola-as-code/blob/3ccb3951ed0752ae9e0e7386a3186e3a05c302ce/docs/telemetry.md) subore, budu na to odkazovat runbooks, pre podrobnosti.